doc/help/elektra-algorithm.md

```diff
@@ -138,7 +138,7 @@ to determine if they are responsible for a key or not.	Consequently, it
 can happen that more than one backend delivers a key with the same name.
 
 `kdbGet()` ensures that a key is uniquely identified by its name.
-Elektra's core will pop keys that are outside of the backend's
+Elektra's core will drop keys that are outside of the backend's
```

Comment:
Its better to keep `pop` and put a link to the glossary, because `pop`  is also the term used in the API.